bundle/builder.go

```diff
@@ -0,0 +1,67 @@
+package bundle
+
+import (
+	"bytes"
+	"context"
+	"io/fs"
+	"testing/fstest"
+
+	layerfs "github.com/dschmidt/go-layerfs"
```

Comment:
Not a lot of stars but I think if it gets the job done we can use it

Reply:
This is used solely to merge the FS, without doing any changes to the underlying FS.
Due to the aforementioned comment regarding the roadmap of these changes, I think that it can be enough for now.
What do you reckon? @eyalkraft

Reply:
Since you went and explored the code we can use it 👍